lib/utils/__tests__/isStandardSyntaxDeclaration.test.js

```diff
@@ -138,17 +138,30 @@ describe('isStandardSyntaxDeclaration', () => {
 	it('less map', () => {
 		expect(isStandardSyntaxDeclaration(lessDecl('@map: { key: value; }'))).toBe(false);
 	});
+
 	it('scss map declaration', () => {
 		expect(isStandardSyntaxDeclaration(scssDecl('$foo: (key: value, key2: value2)'))).toBe(false);
 	});
+
 	it('scss map declaration with quotes', () => {
 		expect(isStandardSyntaxDeclaration(scssDecl('$foo: ("key": value, "key2": value2)'))).toBe(
 			false,
 		);
 	});
+
 	it('scss list declaration', () => {
 		expect(isStandardSyntaxDeclaration(scssDecl('$foo: (value, value2)'))).toBe(false);
 	});
+
+	it('scss pure value in parentheses', () => {
+		expect(isStandardSyntaxDeclaration(scssDecl('a { height: (0) }'))).toBe(true);
+	});
+
+	it('scss complex value in parentheses', () => {
+		expect(isStandardSyntaxDeclaration(scssDecl('a { height: (3px + 5px) * (2px + 4px) }'))).toBe(
+			true,
+		);
+	});
```

Comment:
@ekumlin Thanks for creating this PR! 😄 

The added test cases do not seem to cover the new logic...? 🤔 

```console
$ npx jest lib/utils/__tests__/isStandardSyntaxDeclaration.test.js --coverage
...

$ open .coverage/lcov-report/lib/utils/isStandardSyntaxDeclaration.js.html
```

<img width="638" alt="image" src="https://user-images.githubusercontent.com/473530/139605971-0a7aedf3-3631-4930-a606-bb03114566f2.png">

Reply:
That's... very strange. The tests succeed, but when I remove the change and run the tests, the tests fail, so I can't really think of why this might be the case. I'll see if I can figure it out.

Reply:
Actually, on looking at that closer, the new logic *is* tested. What doesn't seem to be tested is the path where prop *does* start with `$`. However, I'd expect that to be covered by this:

![image](https://user-images.githubusercontent.com/1350074/139609273-43396c96-b626-4338-8570-960e91267531.png)

So will take a look at that.

Reply:
Oh, I see. Before the change, there is this block:

![image](https://user-images.githubusercontent.com/1350074/139609437-f7b8e0d4-1d68-478b-ba29-bd769b04ca73.png)

That already checks if the prop starts with `$`. Which means that in any event when the prop is a variable, we already return `false`, so this second block shouldn't even be needed. And indeed, if I remove that block entirely, the tests pass.